### ISSUE TITLE
[Feature Request] Fill the designer developer gap

### ISSUE THREAD

**Issue by Greghster** [matt-curtis/Fluid-for-Sketch#108]
Currently, we can create constraints and preview them swell as export images from it.
This is great for the design process but what is sad is that these properties cannot be exported or viewed like https://github.com/utom/sketch-measure from https://github.com/utom. 

It would be great to combine what he has done interns of filling that gap between a design and developer but show constraint values @utom

**Comment by Greghster** [comment]
Any thoughts on this?

**Comment by matt-curtis** [comment]
Yes - it's been suggested before, and I 100% agree it needs to happen, I just haven't started on it yet. It is coming though :)

**Comment by Greghster** [comment]
Can I give you a mock-up of what I have in mind? That preview window you have + measurements  in a format developers can access

**Comment by matt-curtis** [comment]
What I was thinking was along the lines of Sketch Measure/[Zeplin](https://zeplin.io/), as both are very close to the idea I have in mind.

**Comment by Greghster** [comment]
Yes sketch measure format would be great with constraint inspector and the features of your preview window. Locally hosted index file would be great as we can't use cloud systems
